module/VuFindApi/src/VuFindApi/Controller/ApiTrait.php

```diff
@@ -202,4 +226,39 @@ protected function output($data, $status, $httpCode = null, $message = '')
         $headers->addHeader($contentTypeHeader);
         return $response;
     }
+
+    /**
+     * Init API key settings
+     *
+     * @param array $settings API key settings from config.ini
+     *
+     * @return void;
+     */
+    protected function initApiKeySettings(array $settings): void
+    {
+        $this->developerSettingsService = $this->getService(DeveloperSettingsService::class);
+        $this->apiKeyHeaderField = $settings['header_field'] ?? $this->apiKeyHeaderField;
```

Comment:
Maybe cleaner and ensures that an empty string doesn't override the default?
```suggestion
        if ($field = $settings['header_field'] ?? null) {
            $this->apiKeyHeaderField = $field;
        }
```